prov/sockets/src/sock_progress.c

```diff
@@ -1403,10 +1403,10 @@ static int sock_pe_process_rx_send(struct sock_pe *pe, struct sock_rx_ctx *rx_ct
 			return 0;
 		len += SOCK_CQ_DATA_SIZE;
 	}
-
-	if (pe_entry->done_len == len && !pe_entry->pe.rx.rx_entry) {
-		data_len = pe_entry->msg_hdr.msg_len - len;
-
+	
+	data_len = pe_entry->msg_hdr.msg_len - len;
+	
+	if (pe_entry->done_len == len && !pe_entry->pe.rx.rx_entry) {	
```

Comment:
nit - please watch introducing unneeded white space, especially at the end of a line.  Some patch tools will reject patches containing this.